src/tecnodev/subCategory/SubCategory.java

```diff
@@ -46,6 +49,10 @@ public String getDescription() {
         return description;
     }
 
+    public boolean hasDescription(){
+        return !description.isBlank();
```

Comment:
Acho que seria uma boa verificar aqui se a descrição também não é nula, Arthur :)
```suggestion
        return description != null && !description.isBlank();
```